code/modules/clothing/clothing.dm

```diff
@@ -376,6 +376,13 @@
 	if(get_armor().has_any_armor() || (flags_cover & (HEADCOVERSMOUTH|PEPPERPROOF)))
 		. += span_notice("It has a <a href='?src=[REF(src)];list_armor=1'>tag</a> listing its protection classes.")
 
+	//MONKESTATION ADDITION START - Denotes some clothing traits when examining a clothing piece.
+	if(clothing_traits)
+		if(TRAIT_CHUNKYFINGERS in clothing_traits)
```

Comment:
I think this should just be a `HAS_TRAIT` check instead, as im not sure `clothing_traits` is meant to be directly checked.

Reply:
For context for anyone looking at this: I discussed this with @wraith-54321 on Discord.

Basically: I didn't think `HAS_TRAIT` would work here. To quote my explanation:

> `HAS_TRAIT` checks a field named `_status_traits`, which is a list of currently applied traits
> however, the traits listed in `clothing_traits` are never applied to the clothing item itself - only to the wearer

There is also no `HAS_CLOTHING_TRAIT` macro which might apply here.

After describing all this to Wraith, they said what I have should work.